### PR TITLE
do not add bands info to udm images

### LINKED ISSUE #6: Planet UDM assets should not have the bands object
I've been using the updated planet conversion stuff, and it's working great for the most part. But it looks like its sticking a 'bands' object into the UDM asset. Though there are some bands in there, they're not EO bands (at some point we could perhaps use the raster extension to describe the UDM bands a bit more). So it'd be good to just not include any bands object on the UDM assets, as they're confusing.

[20201211_103040_ssc12_u0001.txt](https://github.com/stactools-packages/planet/files/6697634/20201211_103040_ssc12_u0001.txt)

### PR DESCRIPTION
**Pull request by volaya**
fixes #6

### CODE REVIEW

**Review by gadomski — APPROVED** [review]

**Comment by gadomski** [comment]
LGTM. @volaya I've given you write access to the repo so I'll let you merge, since you most likely will be working on the code her more than I will be.

**Comment by volaya** [comment]
Awesome, thanks a lot!

**Pull request merged**
